Dockerfile

```diff
@@ -12,5 +12,10 @@ WORKDIR /app
 RUN getent group ${GID} || groupadd -g ${GID} dev
 RUN getent passwd ${UID} || useradd -m -u ${UID} -g ${GID} -s /bin/bash dev
 RUN pip install poetry
+
+RUN apt-get update && apt-get install -y jq
```

Comment:
Ah! I think this is partially my fault. You should be able to run the script directly on your local machine, not through docker. I'm sorry that this wasn't clear in the instructions.